src/main/scala/com/redis/Operations.scala

```diff
@@ -180,4 +180,14 @@ trait Operations { self: Redis =>
   // Flushes all the previously watched keys for a transaction
   def unwatch(): Boolean =
     send("UNWATCH")(asBoolean)
+
+  // CONFIG GET
+  def getConfig(key: Any = "*")(implicit format: Format): Option[Map[Option[String], Option[String]]] =
```

Comment:
should we have the return type as `Option[Map[String, Option[String]]]` ? I think `None` as a key of the `Map` doesn't make much sense.